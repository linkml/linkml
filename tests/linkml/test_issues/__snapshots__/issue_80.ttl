@prefix biolink: <https://w3id.org/biolink/vocab/> .
@prefix ex: <http://example.org/> .
@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .

<http://example.org/person/17> a ex:PERSON ;
    biolink:age 43 ;
    biolink:name "Fred Jones" .

