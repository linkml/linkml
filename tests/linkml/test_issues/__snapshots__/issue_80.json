{
   "id": "http://example.org/person/17",
   "name": "Fred Jones",
   "age": 43
}